pysr/sr.py

```diff
@@ -1989,11 +2004,70 @@ def get_hof(self):
             return ret_outputs
         return ret_outputs[0]
 
+    def plot(self, ax, threshold=0.05, fontsize=10):
+        """Create a plot of the best equations.
+
+        Parameters
+         ----------
+        threshold : float
+             The plotting threshold. Plots only equations that with a score
+             higher than threshold*max_score
+        ax : Axes
+             The axes to draw to
```

Comment:
It should automatically create a plot and axes if not passed. i.e., `ax=None` should be the default.